src/plugins/elasticsearch/lib/expose_client.js

```diff
@@ -78,8 +78,8 @@ module.exports = function (server) {
   server.expose('ElasticsearchClientLogging', ElasticsearchClientLogging);
   server.expose('client', client);
   server.expose('createClient', createClient);
-  server.expose('callWithRequestFactory', callWithRequest);
-  server.expose('callWithRequest', callWithRequest(noAuthClient));
+  server.expose('callWithRequestFactory', _.partial(callWithRequest, server));
```

Comment:
This refactor looks good to me. It is used here: https://github.com/elastic/x-plugins/blob/master/kibana/plugins/security/server/lib/get_client_shield.js#L11